cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionDiffCommand.java

```diff
@@ -213,7 +256,37 @@ public void execute() throws CommandException {
     }
   }
 
-  void executeUnsafe() throws CommandException {
+  private void checkMaxStringsBlock(ExtractionDiffPaths extractionDiffPaths)
+      throws MissingExtractionDirectoryExcpetion, CommandException {
+    if (!this.skipMaxStringsBlock
+        && !Strings.isNullOrEmpty(this.pushToBranchName)
+        && !this.pushToBranchName.equals(MASTER_BRANCH_NAME)) {
```

Comment:
The check for the master branch name should not exist - as various repos might have different names for it (master, main etc.).

Also - in line with the comment re the default behaviour - this will be fully controlled by whoever calls the CLI - and they can just not provide this param for the workflow, when it is being run on their version of a master branch.